v2/internal/testcommon/vcr/redact.go

```diff
@@ -131,6 +132,18 @@ func hidePasswords(s string) string {
 	return passwordMatcher.ReplaceAllLiteralString(s, "\"{PASSWORD}\"")
 }
 
+var (
+	secretMatcher  = regexp.MustCompile(`"([A-Za-z]+)?[Ss]ecret":".*"`)
```

Comment:
Are we sure this doesn't match other fields that aren't secrets and don't need to be redacted? Should we be looking specifically for `clientSecret`?

Also did the `keyMatcher` not work for this?

Reply:
keyMatcher does not work for this. I've added a comment and and updated the regex to match `([A-Za-z]+)[Ss]ecret":".*`. Let me know wdyt about that?